examples/stable_diffusion_v2/ldm/modules/attention.py

```diff
@@ -142,6 +162,34 @@ def rearange_in(x):
         k = rearange_in(k)
         v = rearange_in(v)
 
+        if self.enable_flash_attention and q.shape[1] % 16 == 0 and k.shape[1] % 16 == 0:
```

Comment:
is there a reason this condition is not judged in init already? i.e. self.attention = XA if not enable_fa else FA?

Reply:
Here is a dynamic judgment based on the tensor of the actual operation